shared/more/log-send/index.js

```diff
@@ -0,0 +1,34 @@
+// @flow
+import React, {Component} from 'react'
+import {connect} from 'react-redux'
+import Render from './render'
+import logSend from '../../native/log-send'
+
+type State = {
+  logSendId: ?string
+}
+
+class LogSend extends Component<void, {}, State> {
+  state: State;
+
+  constructor (props: {}) {
+    super(props)
+    this.state = {
+      logSendId: null
+    }
+  }
+
+  render () {
+    const onLogSend = () => {
+      logSend().then(logSendId => this.setState({logSendId}))
+    }
+
+    return <Render logSendId={this.state.logSendId} onLogSend={onLogSend} />
+  }
+
+  static parseRoute () {
+    return {componentAtTop: {title: 'Log Send'}}
+  }
+}
+
+export default connect()(LogSend)
```

Comment:
I don't think it needs to be, I probably copied some code somewhere that had that.